### PR TITLE
feat: onboard dotnet samples to m365 gallery

### PR DESCRIPTION
**Pull request by tecton**
Related task: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/25665470

### CODE REVIEW

**Review by summzhan — APPROVED** [review]

File: stocks-update-notification-bot-dotnet/assets/thumbnail.png
Comment (Moazzem-Hossain-pixel) [inline stocks-update-notification-bot-dotnet/assets/thumbnail.png:1]: Asking your kind Cooperation for this regard.

File: stocks-update-notification-bot-dotnet/assets/sample.json
Comment (Moazzem-Hossain-pixel) [inline stocks-update-notification-bot-dotnet/assets/sample.json:1]: Nice

File: basic-blazor-tab-app/assets/thumbnail.png
Comment (Moazzem-Hossain-pixel) [inline basic-blazor-tab-app/assets/thumbnail.png:1]: Fine

Comment (Moazzem-Hossain-pixel) [inline basic-blazor-tab-app/assets/thumbnail.png:1]: Good

File: basic-blazor-tab-app/assets/sample.json
Comment (Moazzem-Hossain-pixel) [inline basic-blazor-tab-app/assets/sample.json:1]: Helpfull

**Pull request merged**